File: 241210/숫자 사이의 배수/multiplication-between-numbers.js
const fs =  require('fs');

let [a,b] = fs.readFileSync(0).toString().trim().split(" ").map(i=>Number(i));

let Sum=0;
let Cnt=0
for (let i=a; i<=b; i++) {
    if (i%5==0 || i%7==0) {
        Sum+=i
        Cnt++
    }
}

console.log(Sum,(Sum/Cnt).toFixed(1))

File: 241210/숫자 사이의 배수/multiplication-between-numbers.py
a, b = map(int, input().split());

Sum=0
Cnt =0
for i in range(a,b+1) :
    if i%5==0 or i%7==0:
        Sum+=i
        Cnt+=1

print(f"{Sum} {(Sum/Cnt):.1f}")